### ISSUE TITLE
Epic: Create new UI components

### ISSUE THREAD

**Issue by OlgaBilogurova** [hackforla/VRMS#374]
### Overview

Create new UI elements (based on finalized 0.4 Style Guide) and add them to UI Kit.

Elements that should be created are inside yellow borders.

![el-1](https://user-images.githubusercontent.com/11681030/97739745-7f212b80-1a9d-11eb-8c2a-0cef2f182a9d.png)
![el-2](https://user-images.githubusercontent.com/11681030/97739750-821c1c00-1a9d-11eb-8734-9af0c1cfad75.png)

Find mockups here => [Finalized v0.4 Style Guide (Figma)](https://www.figma.com/file/9c6pxjImN8MjHAZLH7zxYm/?node-id=23471%3A0)

### Action Items

- [x] PR #373 should be merged before start this task

**This is an Epic task that broken down into a number of smaller tasks:**

- [x] Create checkboxes as a new UI element & reusable component #384

- [x] Add inputs for current skills and wireframing #479

- [x] Create a radio button as a new UI element #469

- [x] Create a progress bar as a new UI element & reusable component #429

- [x] Create a dropdown as a new UI element #501

- [x] Add a search input to existing inputs #488

### Resources/Instructions
This issue the next step after #360 and #371
[Finalized v0.4 Style Guide (Figma)](https://www.figma.com/file/9c6pxjImN8MjHAZLH7zxYm/?node-id=23471%3A0)
Dev UI Kit http://localhost:3001/dev-ui-kit

**Comment by OlgaBilogurova** [comment]
Created a new task #429

**Comment by OlgaBilogurova** [comment]
Created a new task #479

**Comment by OlgaBilogurova** [comment]
Created a new task 'Create search input as a new UI element' #488

**Comment by OlgaBilogurova** [comment]
Created a new task #501
